stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    #modelnoSMOTE=make_pipeline(
</pre><p>Jennifer Ding:</p><ul><li>Probably best not to include commented out code (e.g. cell 125)</li></ul><p>Joe Cerniglia:</p><p>That is an excellent idea.&nbsp;I will do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>